### PR TITLE
Fix incorrect start after SDL crash

### PR DESCRIPTION
**Pull request by OHerasym**
### PR Changes:
- Fix incorrect start after SDL crash

Related: [APPLINK-27204](https://adc.luxoft.com/jira/browse/APPLINK-27204)

### CODE REVIEW

**Comment by OHerasym** [comment]
Please review: @anosach-luxoft , @LuxoftAKutsan , @orhan-mehmedov, @dtrunov

**Comment by LuxoftAKutsan** [comment]
Reviewed

**Comment by dtrunov** [comment]
@OHerasym Reviewed.

**Comment by anosach-luxoft** [comment]
Reviewed

File: StopSDL.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ function shutdown_sdl {
 
 function kill_sdl {
     echo "Kill SDL process : "$pid
-    kill -SIGTERM $pid
+    kill -SIGABRT $pid
```
Comment (orhan-mehmedov) [inline StopSDL.sh]: @OHerasym why not SIGKILL? Is the intention to generate core dump?

**Comment by orhan-mehmedov** [comment]
@OHerasym Reviewed

File: StopSDL.sh (hunk 1 above)
Comment (OHerasym) [inline StopSDL.sh]: @orhan-mehmedov ,  it can be SIGKILL too, I changed from SIGTERM, because SDL catch this signal

**Pull request merged**